--- argparserbase.py	2020-04-17 22:29:22.727747000 -0700
+++ argparserbase.py_fixed	2020-04-17 20:58:20.273941000 -0700
@@ -325,7 +325,7 @@
 
             # Write pid file before chuid
             if options.pidfile:
-                with open(options.pidfile, 'wb') as _file:
+                with open(options.pidfile, 'w') as _file:
                     _file.write('%d\n' % os.getpid())
 
             if not common.windows_check():
